code/game/objects/items/props/helmetgarb.dm

```diff
@@ -496,6 +496,32 @@
 	desc = "The USCM had its funding pulled for these when it became apparent that not every deployed enlisted was wearing a helmet 24/7; much to the bafflement of UA High Command."
 	icon_state = "helmet_gasmask"
 
+/obj/item/prop/helmetgarb/helmet_gasmask/on_enter_storage(obj/item/storage/internal/HelmetInternalInv)
+	if(!istype(HelmetInternalInv))
+		return
+
+	var/obj/item/Helmet = HelmetInternalInv.master_object
+
+	if(!istype(Helmet, /obj/item/clothing/head/helmet))
+		return
+
+	Helmet.flags_inventory = initial(Helmet.flags_inventory)+BLOCKGASEFFECT
+	Helmet.flags_inv_hide = initial(Helmet.flags_inv_hide)+HIDEFACE
+	..()
+
+/obj/item/prop/helmetgarb/helmet_gasmask/on_exit_storage(obj/item/storage/internal/HelmetInternalInv)
```

Comment:
Use snake_case; not PascalCase.

See https://github.com/cmss13-devs/cmss13/blob/master/.github/guides/STYLES.md#type-paths-must-be-snake-case